postinstall.js

```diff
@@ -6,17 +6,22 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  */
 'use strict';
+const path = require('path');
+const fs = require('fs');
+
+let notNPM = true;
```

Comment:
Usually I would implement this as follows:

```js
const isNPMPackage = () => {
  try {
    …
    return true;
  } catch (e) {}
  return false;
};

// later
if (!isNPMPackage()) {

} 
```

This way you don't have any state (no variables) and the functionality is self-contained. In older versions of v8 it would also not de-opt the rest of the file because of the try-catch (not that it matters here, just a general guideline :) ).